pkgs/development/tools/nimble/default.nix

```diff
@@ -12,22 +12,22 @@ stdenv.mkDerivation rec {
     sha256 = "12znxzj1j5fflw2mkkrns9n7qg6sf207652zrdyf7h2jdyzzb73x";
   };
 
-  buildInputs = [ nim ];
+  buildInputs = [ nim openssl ];
 
   patchPhase = ''
     substituteInPlace src/nimble.nim.cfg --replace "./vendor/nim" "${nim}/share"
+    cat >>src/nimble.nim.cfg <<END
+--clib:crypto
```

Comment:
I don't like the indentation of this. Wouldn't `echo "--clib:crypto" > src/nimble.nim.cfg` be enough here?